test/test-loginController.js

```diff
@@ -125,6 +125,23 @@ describe('loginController', function() {
           done();
         })
       });
+
+      it('should say that email is invalid', function(done) {
+        chai.request(app)
+        .post('/psychologue/login')
+        .type('form')
+        .send({
+          'email': 'fake_it',
+        })
+        .redirects(0)
+        .end((err, res) => {
+          sinon.assert.notCalled(getPsychologistByEmailStub);
```

Comment:
@estellecomment une idée de comment tester le contenu du `req.flash` par hasard ?

en pseudo code
```
res.text.contains("Vous devez spécifier un email valide.")
```

Reply:
la derniere fois que j'ai regardé (sur audioconf) j'ai pas vraiment réussi... Du coup depuis j'ai pas essayé... C'est vrai que ca pourrait etre cool (au moins tester la presence de messages d'erreur/succes par exemple)
Donc pour le moment j'ai zappé.

Reply:
j'ai tenté, et pas réussi non plus

L'objet `res.locals` où est stocké `req.flash` n'est pas disponible avec chai-http 

https://stackoverflow.com/questions/42394691/how-to-test-a-property-on-the-res-locals-object#comment71938479_42394691